src/main/kotlin/ar/edu/unahur/obj2/vendedores/Centros.kt

```diff
@@ -0,0 +1,22 @@
+package ar.edu.unahur.obj2.vendedores
+
+class Centros(val ciudad: Ciudad){
+
+    val vendedoresDelCentro = mutableListOf<Vendedor>()
+
+    fun agregarVendedor(unVendedor : Vendedor){
+        if(!vendedoresDelCentro.contains(unVendedor) ){
+            vendedoresDelCentro.add(unVendedor)
+        }else{
+            throw Exception("El Vendedor ya se encuentra trabajando con este centro.")
+        }
```

Comment:
Está muy bien, aunque resulta más fácil de leer si hacés la validación arriba. Haciendo así, se puede incluso omitir el `else`:

```suggestion
        if(vendedoresDelCentro.contains(unVendedor) ){
            throw Exception("El Vendedor ya se encuentra trabajando con este centro.")
        }
        vendedoresDelCentro.add(unVendedor)        
```